### DISCUSSION TITLE
FaceResult

### DISCUSSION THREAD

**Discussion by pgg** [vladmandic/human#228] [Q&A] [answered]
Any specific reason why gender, emotion and race is not typed? I can't see what their values could be from the docs, think it used to be there, but not anymore, or am I looking in the wrong place?

**Comment by vladmandic** [comment, marked as answer]
No reason, I just haven't been tidy with strong typing string enums until recently  
I've just added typedefs for all remaining string enums in `results`:

```js
export type { Emotion, Finger, FingerCurl, FingerDirection, HandType, Gender, Race, FaceLandmark, BodyLandmark, BodyAnnotation, ObjectType } from './result';
```

**Reply by pgg** [reply]
ok, thanks
